style/base.css

```diff
@@ -12,6 +12,29 @@
   flex-grow: 1;
 }
 
+.glue-Control-datasets-container {
+  padding: 0;
+  margin: 0;
+}
+
+.glue-Control-datasets-item {
+  display: flex;
+  flex-direction: row;
+  color: var(--jp-ui-font-color1);
+  height: 24px;
+  line-height: 24px;
+  padding: 0 8px;
+  cursor: pointer;
+}
+
+.glue-Control-datasets-item:hover {
+  background-color: var(--jp-layout-color2);
```

Comment:
Maybe the color could be different when selected and hover.

Reply:
Thanks! I looked into what the filebrowser widget does for selections and applied the same styling.

![Screenshot from 2023-04-18 11-01-44](https://user-images.githubusercontent.com/21197331/232728285-db102a77-97fa-46a6-bb44-196b2ff4a44f.png)